db/schema.rb

```diff
@@ -15,12 +15,12 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "applications", force: :cascade do |t|
+  create_table "applications", id: :serial, force: :cascade do |t|
     t.string "name"
     t.string "email"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.bigint "event_id"
+    t.integer "event_id"
```

Comment:
The schema has changed, was there a migration that ran but you removed later? If not, could you just delete remove the schema from the PR?

Reply:
Ah yes, I ran db:migrate a few times while I was dealing with the wrong database credentials. No new migrations though so I will delete the file. Thank you :)